src/rmkit/input/input.cpy

```diff
@@ -59,30 +62,11 @@ namespace input:
           #ifdef DEBUG_INPUT_EVENT
           fprintf(stderr, "\n")
           #endif
-          event = {} // clear the event?
+          prev_ev = event
+          event = prev_ev
```

Comment:
I think the `event = prev_ev` part is redundant? Also should this have a call to event.finalize() somewhere?

Reply:
event.finalize() is called on `EV_SYN`

Reply:
oh you're totally right, it's like 5 lines above this :facepalm: